# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.cpp
# Commit: 95b350572bbf
# Full Hash: 95b350572bbf079fdf6613b5a721c6ee5023fc6e
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1646563
# File Overlap Count: 1
# Description:
#   Bug 1646563 - Remove BrowserId from nsFrameLoaderOwner, r=nika
#   
#   We can just use BrowsingContext::BrowserId directly, so it's unnecessary to have
#   the field on nsFrameLoaderOwner as well.
#   
# ==============================================================================

diff -r 4eab38aba3fe -r 95b350572bbf dom/ipc/ContentChild.cpp
--- a/dom/ipc/ContentChild.cpp	Mon Jun 22 21:30:31 2020 +0000
+++ b/dom/ipc/ContentChild.cpp	Mon Jun 22 21:46:03 2020 +0000
@@ -949,9 +949,8 @@
     openerBC = parent;
   }
 
-  uint64_t browserId(nsContentUtils::GenerateBrowserId());
   RefPtr<BrowsingContext> browsingContext = BrowsingContext::CreateDetached(
-      nullptr, openerBC, aName, BrowsingContext::Type::Content, browserId);
+      nullptr, openerBC, aName, BrowsingContext::Type::Content);
   MOZ_ALWAYS_SUCCEEDS(browsingContext->SetRemoteTabs(true));
   MOZ_ALWAYS_SUCCEEDS(browsingContext->SetRemoteSubframes(useRemoteSubframes));
   MOZ_ALWAYS_SUCCEEDS(browsingContext->SetOriginAttributes(